src/MyElefant.php

```diff
@@ -215,8 +216,58 @@ public function sendSms($campaignId, $campaignName, $contacts, $sendDate = null,
                     'info',
                     MyElefantConfig::SUCCESS_MESSAGE . ', ' . MyElefantConfig::CREDIT_REMAINING . ' ' . $arr_body->solde
                 );
+                return true;
+            }
+        }
+        return false;
+    }
+
+    /**
+     * Sending sms
+     *
+     * @param string $campaignId Campaign's id
+     * @param array  $content
+     *
+     * @throws @Exception
+     * @return bool
+     */
+    public function sendSms($campaignId, $content)
+    {
+        if (!$this->checkSendSmsContentFormat($content) || !$this->checkPhoneNumber($content[0])) {
+            $this->setLog(
+                'critical',
+                MyElefantConfig::CRITICAL_MESSAGE_CONTENT_FORMAT
+            );
+            throw new Exception($content . ' ' . MyElefantConfig::CRITICAL_MESSAGE_CONTENT_FORMAT);
+        }
+        try {
+            $client   = new Client();
+            $response = $client->request(
+                'POST',
+                MyElefantConfig::URL_MYELEFANT_API . MyElefantConfig::URL_MYELEFANT_API_SEND_SMS,
+                [
+                    'headers' => [
+                        'Authorization' => $this->token,
+                        'Content-Type'  => 'application/json'
+                    ],
+                    'json'    => [
+                        'campaign_uuid' => $campaignId,
+                        'contact'       => $content
+                    ]
+                ]
+            );
+        } catch (\Throwable $e) {
+            $this->setLog('critical', $e->getMessage());
+            throw new Exception($e->getMessage());
+        }
+        if ($response->getStatusCode() == 200) {
+            $body     = $response->getBody();
+            $arr_body = json_decode($body);
+            if ($arr_body->success == true) {
+                return true;
             }
```

Comment:
You should **always** use strict comparison operators (`===`) unless you have a very good reason not do so.
In this case, you can even simplify the return:
```php
return $arr_body->success === true;
```
or
```php
return (bool)$arr_body->success;
```